discovery.go

```diff
@@ -198,7 +198,13 @@ func (s *PHPStore) discoverPHPViaPHP(dir, binName string) *Version {
 		FullVersion: v,
 		PHPPath:     php,
 	}
-	fpm := filepath.Join(dir, "sbin", strings.Replace(binName, "php", "php-fpm", 1))
+	fpm := filepath.Join(dir, "bin", strings.Replace(binName, "php", "php-fpm", 1))
+	fpmSBin := filepath.Join(dir, "sbin", strings.Replace(binName, "php", "php-fpm", 1))
```

Comment:
Can you inline this directly? No need to create a temp variable that might not be used.

Reply:
like so? 😅